UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Program.cs

```diff
@@ -159,6 +158,7 @@ private static void ConfigureServices(IServiceCollection serviceCollection, ICon
 
             serviceCollection.AddDistributedMemoryCache();
 
+            serviceCollection.AddSingleton<IConfiguration>(configuration);
```

Comment:
WE are using the configuration object on this line where as we have checked for a null value on line 148. If the configuration was null then this could be an issue?

Why is the null check completed above but not on this line?

Coverity has flagged this as an issue!

Reply:
Resolved